crates/rust-analyzer/src/global_state.rs

```diff
@@ -192,6 +192,7 @@ impl GlobalState {
                 if let Some(path) = vfs.file_path(file.file_id).as_path() {
                     let path = path.to_path_buf();
                     if reload::should_refresh_for_change(&path, file.change_kind) {
+                        tracing::warn!("fetch-fiel_change");
```

Comment:
Should be removed or changed to `info!`?

Reply:
was already removed in a follow up

Reply:
Yeah, I actually missed it in #12947